src/tests.builds

```diff
@@ -2,12 +2,17 @@
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <ItemGroup>
-    <Project Include="*\test*\**\*.csproj">
+    <Project Include="*\test*\**\*.csproj" Condition="'$(OSEnvironment)'=='Windows_NT'">
       <OSGroup Condition="'$(FilterToOSGroup)'!=''">$(FilterToOSGroup)</OSGroup>
     </Project>
     <Project Include="*\test*\**\*.vbproj" Condition="'$(IncludeVbProjects)'!='false'">
```

Comment:
I'm not sure where IncludeVbProjects gets set but perhaps that is the condition you should use here as well.